examples/supplementary/format_concat_args/.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
This file can be removed. A `.gitignore` in a parent directory accomplishes this.